app/src/main/java/hr/foi/air/servicesync/ui/contents/dujino.txt

```diff
@@ -0,0 +1,103 @@
+
```

Comment:
This file has to be deleted. But it can pass for now, it doesn't interfere with anything.